README.md

```diff
@@ -235,10 +233,10 @@ Would throw the following error, because of the lambda literal argument:
 
 > ⚠️ Arguments of a constant creation must be constant expressions.
 
-Even though, if you'd pass a reference of a _named_ top-level/static function
+Even though if you'd passed a reference of a _named_ top-level/static function
```

Comment:
```suggestion
Even though if you'd pass a reference to a _named_ top-level/static function
```
or
```suggestion
Even though if you would've passed a reference to a _named_ top-level/static function
```
Is correct I think :sweat_smile:

Reply:
Hmm, looking it up, I _think_ the way I currently have it ("if you had") is more "correct"? I think I've heard both ways before, though, so not sure.

Reply:
Ahh yeah "if you had" is correct indeed, currently it says "you'd" with the "'d" :smile:

Then I guess the third option is the following, which is what I think you were going for?
```suggestion
Even though if you passed a reference of a _named_ top-level/static function
```